### DISCUSSION TITLE
Welcome to minesweeper Discussions!

### DISCUSSION THREAD

**Discussion by SprigganCS** [SprigganCS/minesweeper#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ArthurProenca** [comment]
Um bom desafio

**Reply by SprigganCS** [reply]
realmente

**Reply by ArthurProenca** [reply]
sim

**Comment by ArthurProenca** [comment, marked as answer]
Como posso fazer para ele resolver jogos?

**Reply by SprigganCS** [reply]
só mandar por linha de comando usando o operador "<"

**Reply by ArthurProenca** [reply]
ok
